lib/crystalball.rb

```diff
@@ -10,6 +10,7 @@
 require 'crystalball/map_generator'
 require 'crystalball/map_generator/configuration'
 require 'crystalball/map_generator/coverage_strategy'
+require 'crystalball/rails/map_generator/action_view_strategy'
```

Comment:
this should be required in `crystalball/rails` only. If user wants to use Crystalball with rails he'll have to require `crystalball/rails` explicitly to get rails-specific features.